<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>server_gui</class>
 <widget class="QDialog" name="server_gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>20</y>
     <width>501</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="p1_layout">
      <item>
       <widget class="QLabel" name="p1_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Player 1:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="player1">
        <property name="text">
         <string>Leer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="ID1_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>ID: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="p1_kill_button">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Kick</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="p2_layout">
      <item>
       <widget class="QLabel" name="p2_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Player 2:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="player2">
        <property name="text">
         <string>Leer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="ID2_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>ID: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="p2_kill_button">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Kick</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="p3_layout">
      <item>
       <widget class="QLabel" name="p3_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Player 3:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="player3">
        <property name="text">
         <string>Leer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="ID3_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>ID: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="p3_kill_button">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Kick</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="p4_layout">
      <item>
       <widget class="QLabel" name="p4_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Player 4:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="player4">
        <property name="text">
         <string>Leer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="ID4_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>ID: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="p4_kill_button">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Kick</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="startServer_button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Starte Server</string>
   </property>
  </widget>
  <widget class="QPushButton" name="starGame_button">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>290</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Starte Spiel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="abbrechen">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>290</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Abbrechen</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopServer_button">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>290</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop Server</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>251</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="new_game_settings_layout">
    <item>
     <layout class="QVBoxLayout" name="spieler_layout">
      <item>
       <widget class="QRadioButton" name="two_player_btn">
        <property name="text">
         <string>2 Spieler</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="three_player_btn">
        <property name="text">
         <string>3 Spieler</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="four_player_btn">
        <property name="text">
         <string>4 Spieler</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="spielfeld_layout">
      <item>
       <widget class="QLabel" name="spielfeld_label">
        <property name="text">
         <string>Spielfeld</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="hoehe_layout">
        <item>
         <widget class="QLabel" name="hoehe_label">
          <property name="text">
           <string>Höhe</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="hoehe_spinbox">
          <property name="value">
           <number>8</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="breite_layout">
        <item>
         <widget class="QLabel" name="breite_label">
          <property name="text">
           <string>Breite</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="breite_spinbox">
          <property name="value">
           <number>8</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="spielername_label">
   <property name="geometry">
    <rect>
     <x>292</x>
     <y>183</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Spielername</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="spielername_lineEdit">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>180</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>abbrechen</sender>
   <signal>clicked()</signal>
   <receiver>server_gui</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>505</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
